### PR TITLE
RELATED: BB-1690 Support measure value filters in UI SDK

### PR DESCRIPTION
**Pull request by qduval**
---

# PR checklist

- [ ] Change was tested using dev-release in Analytical Designer and Dashboards (if applicable).
- [ ] Change is described in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Migration guide (for major update) is written to [CHANGELOG.md](../blob/master/CHANGELOG.md).

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-oss-check/4/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/424/ : SUCCESS in 2m 51s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-oss-check/5/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/425/ : SUCCESS in 1m 03s

File: src/DataLayer/converters/FilterConverter.ts [hunk 1]
```diff
@@ -60,3 +71,13 @@ export function convertVisualizationObjectFilter(filter: VisualizationObjectFilt
         return convertRelativeDateFilter(filter);
     }
 }
+
+export function convertVisualizationExtendedObjectFilter(
```
Comment (strouhaljan) [inline src/DataLayer/converters/FilterConverter.ts]: This should be called `convertVisualizationObjectExtendedFilter`.

Comment (strouhaljan) [reply src/DataLayer/converters/FilterConverter.ts]: Rename all the occurrences below.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-oss-check/6/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/426/ : SUCCESS in 2m 14s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-oss-check/7/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/427/ : FAILURE in 1m 32s

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,11 @@ The REST API versions in the table are just for your information as the values a
 |\>= 10.0.0|3
 |<= 9.0.1|2
 
+<a name="11.18.3"></a>
```
Comment (strouhaljan) [inline CHANGELOG.md]: Release these changes as minor, not patch.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-oss-check/8/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/428/ : FAILURE in 18s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-merge-compliance-check/96/ : FAILURE in 22s
- gooddata-js-unit-tests-zuul-docker-el7 : SKIPPED

**Comment by qduval** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-oss-check/9/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/429/ : SUCCESS in 3m 23s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-merge-compliance-check/97/ : SUCCESS in 20s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/430/ : SUCCESS in 2m 29s

**Pull request merged**